### PR TITLE
Mouse pointer

### PR DESCRIPTION
**Pull request by a19steis**
Fixed mouse pointer when hovering over mouse mode buttons, also for the "+" sign.

### CODE REVIEW

File: DuggaSys/diagram.php [hunk 1]
```diff
@@ -464,14 +464,14 @@
...
                 </fieldset>
 
-                <input type="button" value="Mouse Selection" onclick='setMouseMode(0);' />
-                <input type="button" value="Entity" onclick='setMouseMode(1);' />
-                <input type="button" value="Relation" onclick='setMouseMode(2);' />
-                <input type="button" value="Attribute" onclick='setMouseMode(3);' />
+                <input class="paneButtons" value="Mouse Selection" onclick='setMouseMode(0);' />
+                <input class="paneButtons" value="Entity" onclick='setMouseMode(1);' />
+                <input class="paneButtons" value="Relation" onclick='setMouseMode(2);' />
+                <input class="paneButtons" value="Attribute" onclick='setMouseMode(3);' />
```
Comment (a19chrel) [inline DuggaSys/diagram.php:474]: This change removes the input-type "button" that makes the input go to the default type value of "text". Therefore it is possible to edit the field.

**Review by a19chrel — CHANGES_REQUESTED** [review]
While adding the class to the input fields, the type was removed and makes the input-fields to be text-fields and not buttons. This makes the fields editable.

File: DuggaSys/diagram.php (hunk 1 above)
Comment (a19steis) [reply DuggaSys/diagram.php]: Correct, added button types.

**Review by a19chrel — APPROVED** [review]
LGTM

**Pull request merged**